### PR TITLE
Revert "Revert "Removes fusion""

### PR DESCRIPTION
**Pull request by Seris02**
Reverts Skyrat-SS13/Skyrat13#2534

Reasons for no fusion:

1. Fusion doesn't really give you anything you need, atmospherics or gas wise.
2. Fusion usually ends up fucking the station.
3. Shitters use fusion to fuck up the station.
## 4. Fusion literally has no use on the station other than screwing it up. It doesn't generate research or anything like that, literally all it does is make heat that you can't do anything with because it's too hot.

Cargo doesn't ever get any fusion canisters because atmos technicians don't bother to give it to them, so don't give that excuse.

# literally no reason to keep fusion.

### CODE REVIEW

**Comment by ForrestWick** [comment]
We should prolly give atmos some shit to do that's fun in exchange at least. I see no reason to full on delete fusion, I think instead we should just make it less outwardly shitty and server-killing

**Comment by davidchan13** [comment]
Fusion was removed for valid reasons, especially when you consider it didn't take any special equipment to start a fusion reaction. A common air canister could house a fusion reaction and put off more heat and rads than a small undergoing delam

**Comment by Akrilla** [comment]
Fusion was removed for server stability, now that we're in a better spot this removal would be for gameplay reasons. Hard disagree on it needing to be removed personally.

**Comment by Yawet330** [comment]
> Reverts #2534
> 
> Reasons for no fusion:
> 
> 1. Fusion doesn't really give you anything you need, atmospherics or gas wise.
> 2. Fusion usually ends up fucking the station.
> 3. Shitters use fusion to fuck up the station.
> 
> ## 4. Fusion literally has no use on the station other than screwing it up. It doesn't generate research or anything like that, literally all it does is make heat that you can't do anything with because it's too hot.
> Cargo doesn't ever get any fusion canisters because atmos technicians don't bother to give it to them, so don't give that excuse.
> 
> # literally no reason to keep fusion.

you literal tool, fusion can be used to generate research points via multitooling a radiation collector, and adding a tank of trit to it. and its very fucking fast too (it scales with rad level).

**Comment by davidchan13** [comment]
> > Reverts #2534
> > Reasons for no fusion:
> > 
> > 1. Fusion doesn't really give you anything you need, atmospherics or gas wise.
> > 2. Fusion usually ends up fucking the station.
> > 3. Shitters use fusion to fuck up the station.
> > 
> > ## 4. Fusion literally has no use on the station other than screwing it up. It doesn't generate research or anything like that, literally all it does is make heat that you can't do anything with because it's too hot.
> > Cargo doesn't ever get any fusion canisters because atmos technicians don't bother to give it to them, so don't give that excuse.
> > # literally no reason to keep fusion.
> 
> you literal tool, fusion can be used to generate research points via multitooling a radiation collector, and adding a tank of trit to it. and its very fucking fast too (it scales with rad level).

Trit can be produced without fusion.

**Comment by frontline03** [comment]
Fusion produces Pluoxium, A gas BETTER than oxygen as when you put it in a tank and you have it about roughly 1-4 KPA, you will be breathing about 16 KPA oxygen which is what is normally outputted.

**Comment by davidchan13** [comment]
> Fusion produces Pluoxium, A gas BETTER than oxygen as when you put it in a tank and you have it about roughly 1-4 KPA, you will be breathing about 16 KPA oxygen which is what is normally outputted.

Pluoxium is produced by irradiating Oxygen. SM can produce it as can other radiation sources such as the singularity and uranium. Fusion is not required for this. Pluox is 8 times more efficient than O2, requiring 3kPa from a tank to sustain a person.

Fusion does nothing unique except break the temperature and radiation calculations of the game because it has no limiting factors and requires absolutely no specialized equipment like actual real world fusion does. There is no limiting factors and leads to a positive feedback loop until the game runs out of digits and treats the gas as if it has infinite heat.

**Comment by frontline03** [comment]
There's also the fact that Fusion is one of the few things atmos tech can really do that's...fun, Yeah it can go absolutely fucking ham with it irradiating the server, but removing it entirely is just going to piss people off. You should've put forward something like a cap on it.

**Comment by Hybor** [comment]
> > > Reverts #2534
> > > Reasons for no fusion:
> > > 
> > > 1. Fusion doesn't really give you anything you need, atmospherics or gas wise.
> > > 2. Fusion usually ends up fucking the station.
> > > 3. Shitters use fusion to fuck up the station.
> > > 
> > > ## 4. Fusion literally has no use on the station other than screwing it up. It doesn't generate research or anything like that, literally all it does is make heat that you can't do anything with because it's too hot.
> > > Cargo doesn't ever get any fusion canisters because atmos technicians don't bother to give it to them, so don't give that excuse.
> > > # literally no reason to keep fusion.
> > 
> > 
> > you literal tool, fusion can be used to generate research points via multitooling a radiation collector, and adding a tank of trit to it. and its very fucking fast too (it scales with rad level).
> 
> Trit can be produced without fusion.

Perhaps re-read the original post. You have very poor reading comprehension.

**Comment by Hybor** [comment]
Don't remove fusion if you're not going to put in something worthwhile in its place.

**Comment by davidchan13** [comment]
> Perhaps re-read the original post. You have very poor reading comprehension.

My reading comprehension is fine. Fusion does nothing unique you can't already do with controlled burns of plasma and oxygen. Well that's not entirely true. Fusion breaks the game calcs, clogs up the processor and completely imbalances how the gases are used. 

Perhaps you should work on your ability to debate and re-iterate your point, because yours and the entire pro-fusion talking heads in this PR have been shrieking 'MUH FUH-JUN. MUH NUMBAS'

Someone wants to rework fusion, fine. Someone wants to put forward a PR that brings it under control and isn't something that can be done round start that puts out so much radiation and heat that it instant kills ash drakes and husks lings in the same zip code, fine. Fusion has been in Skyrat since the start, Fusion has been on tg for years. The only reason it was never removed was because the coders maintaining it were the ones abusing it. 

TL;DR, you want to keep fusion put up a PR that actually fixes and balances it. Till then, goodbye fusion.

**Comment by frontline03** [comment]
> > Perhaps re-read the original post. You have very poor reading comprehension.
> 
> My reading comprehension is fine. Fusion does nothing unique you can't already do with controlled burns of plasma and oxygen. Well that's not entirely true. Fusion breaks the game calcs, clogs up the processor and completely imbalances how the gases are used.
> 
> Perhaps you should work on your ability to debate and re-iterate your point, because yours and the entire pro-fusion talking heads in this PR have been shrieking 'MUH FUH-JUN. MUH NUMBAS'
> 
> Someone wants to rework fusion, fine. Someone wants to put forward a PR that brings it under control and isn't something that can be done round start that puts out so much radiation and heat that it instant kills ash drakes and husks lings in the same zip code, fine. Fusion has been in Skyrat since the start, Fusion has been on tg for years. The only reason it was never removed was because the coders maintaining it were the ones abusing it.
> 
> TL;DR, you want to keep fusion put up a PR that actually fixes and balances it. Till then, goodbye fusion.

Right go fuck yourself before you compare me to a fucking retard. Secondly, Fusion is one of the few fucking things an atmos tech can actually do, You wanna remove one of the few things they can do thats fun for them? Regarding shitters who use it to destroy the server, blame the player, not fusion, By that logic we should remove botany because I can make explosives in there that can kill people.

**Comment by Azarak** [comment]
included in https://github.com/Skyrat-SS13/Skyrat13/pull/2804
